frontend/src/model/beta/settings/ModelAccess.tsx

```diff
@@ -51,19 +56,23 @@ export default function ModelAccess({ model }: ModelAccessProps) {
   const entities = useMemo(() => {
     if (!users) return []
 
-    const userGroup = users.find((usrGroup) => usrGroup.kind === 'user')
-    if (userGroup) {
-      return userGroup.entities
-    } else {
-      return []
+    const userList = users.find((usrGroup) => usrGroup.kind === 'user')
```

Comment:
We need to review what the backend is returning to avoid:
1. Having to do unnecessary computation on parsing/ sorting the response.
2. Hard-coding Entity Kinds here. Ideally the frontend should just present the information it receives from the backend without needing to know or work-out what Entity Kinds are included in the response.